ukrainian/dr-who/dr-who.md

```diff
@@ -0,0 +1,5 @@
+Це інша планета або інопланетний літальний апарат, отже з мобілочки однозначно можна викликати доктора Хто - саме це я і зроблю. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.